### PR TITLE
removing translator from sapm-proto as its moved to golib

### PR DESCRIPTION
**Pull request by jgheewala**

### CODE REVIEW

File: client/worker.go [hunk 1]
```diff
@@ -29,8 +29,8 @@ import (
 	jaegerpb "github.com/jaegertracing/jaeger/model"
 	"go.opencensus.io/trace"
 
+	"github.com/signalfx/golib/v3/translator"
```
Comment (owais) [inline client/worker.go]: This won't work as it'll still result in a circular dependency. Can we copy this one function (OCStatusCodeFromHTTP) to this package (client) and use it directly here?

Comment (jgheewala) [reply client/worker.go]: and here i thought i fixed it

**Comment by jgheewala** [comment]
@owais moved that to oc_status_code.go

**Comment by owais** [comment]
Thanks. Approving again.

**Comment by owais** [comment]
Released https://github.com/signalfx/sapm-proto/releases/tag/v0.4.0

**Pull request merged**